lib/events/stream.go

```diff
@@ -250,21 +250,18 @@ func NewProtoStream(cfg ProtoStreamConfig) (*ProtoStream, error) {
 	}
 	cancelCtx, cancel := context.WithCancel(context.Background())
 	completeCtx, complete := context.WithCancel(context.Background())
-	uploadsCtx, uploadsDone := context.WithCancel(context.Background())
```

Comment:
These are not necessary as the `receiveAndUpload` needs to complete before any of `Close`/`Complete` exit, and the `uploadLoopDoneCh` carries the same amount of information.